src/models.py

```diff
@@ -12,3 +14,12 @@ class User(Base):  # pylint: disable=too-few-public-methods
 
     id = Column(Integer, primary_key=True)  # noqa: A003
     username = Column(String, nullable=False, unique=True)
+    password_hash = Column(String(64), nullable=False)
+
+    def set_password(self, password: str) -> None:
+        """Use this method to set hashed password to User."""
+        self.password_hash = hashlib.sha256(str(password).encode()).hexdigest()
+
+    def check_password(self, password_to_check: str) -> bool:
+        """Use this method to check User's password."""
+        return hashlib.sha256(str(password_to_check).encode()).hexdigest() == self.password_hash
```

Comment:
Password hashing logic appears twice in `set_password` and in `check_password`. If one day we will need to change password hashing logic we will need to apply changes in two places. It's not good to have logic which has one reason to change to be in more than one place.